dakota/oat/cooling-time/cooling-time-exp.py

```diff
@@ -0,0 +1,39 @@
+import glob
+import subprocess
+import dakota.interfacing as di
+import sys
+import os
+sys.path.append('../../../scripts')
+import input as inp
+
+cycdir = '../../../cyclus-files/oat/cooling-time/'
+
+# ----------------------------
+# Parse Dakota parameters file
+# ----------------------------
+
+params, results = di.read_parameters_file()
+
+# -------------------------------
+# Convert and send to Cyclus
+# -------------------------------
+
+# Edit Cyclus input file
```

Comment:
A comment like this is a good indication that what follows should be in its own function.